docs/guides/setup/NCI/basics.rst

```diff
@@ -66,20 +82,34 @@ Set :guilabel:`Modules` to ``dea``.
    </video>
 
 
-Virtual Desktops with ARE
-=========================
+.. dropdown:: Troubleshooting: fe_sendauth: no password supplied
```

Comment:
Minor thing: If this is a literal error message, can we put it in single quote marks. This is just to avoid having a colon within a colon.

```rst
.. dropdown:: Troubleshooting: 'fe_sendauth: no password supplied'
```